### PR TITLE
Entity: Improved player movement

### PR DESCRIPTION
**Pull request by PeratX**
### Description

Player movement should use MovePlayerPacket to send.

### Tests & Reviews

I have only one device. Need tests

### CODE REVIEW

**Comment by FaanMaario** [comment]
Tested, and there are some issues with the head of players.
Sometimes the head is upside down.

**Comment by dktapps** [comment]
Code Review -1. The ordering of yaw, headYaw and pitch are wrong and the values have also changed as of 0.15.0

File: src/pocketmine/level/Level.php [hunk 1]
```diff
@@ -3206,6 +3199,33 @@ public function addEntityMovement($chunkX, $chunkZ, $entityId, $x, $y, $z, $yaw,
...
+			$this->moveToSend[$index] = [];
+		}
+
+		$pk = new MovePlayerPacket();
+		$pk->eid = $entityId;
+		$pk->x = $x;
+		$pk->y = $y;
+		$pk->z = $z;
+		$pk->yaw = $yaw;
+		$pk->headYaw = $headYaw === null ? $yaw : $headYaw;
```
Comment (dktapps) [inline src/pocketmine/level/Level.php]: This is incorrect. MovePlayerPacket does not have headYaw, it has bodyYaw.

**Comment by PeratX** [comment]
@dktapps Fixed. Please review again.

**Comment by dktapps** [comment]
Seems functional after the latest commit. Now for squashing and then it's good to go

**Comment by PeratX** [comment]
squash??

**Comment by dktapps** [comment]
Yes, by rebasing?

**Comment by XFuryMCPE** [comment]
Was this ever added? Or @PeratX just deleted it..?

**Comment by jojoe77777** [comment]
@XFuryMCPE https://github.com/iTXTech/Genisys/commit/57d4552706305a8f73477a8a41a1a3f9a2ec5846

**Comment by XFuryMCPE** [comment]
Oh sweet @jojoe77777

**Pull request closed without merging**